### PR TITLE
Translate concepts/cluster-administration/cloud-providers/ into Japanese

### LINKED ISSUE #23120: Translate concepts/cluster-administration/cloud-providers/  into Japanese
**This is a Feature Request**

 **What would you like to be added**
Translate the following document into Japanese.
https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by nishipy**
**What this PR does / why we need it**:
This page has not been translated.

Original file is as follows:
https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/cluster-administration/cloud-providers.md

**Which issue(s) this PR fixes:**
Close #23120 .

**Special notes for your reviewer:**
None.

### CODE REVIEW

File: content/ja/docs/concepts/cluster-administration/cloud-providers.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+  extraArgs:
+    cloud-provider: "openstack"
+    cloud-config: "/etc/kubernetes/cloud.conf"
+  extraVolumes:
+  - name: cloud
+    hostPath: "/etc/kubernetes/cloud.conf"
+    mountPath: "/etc/kubernetes/cloud.conf"
+```
+
+インツリークラウドプロバイダーは、通常、[kube-apiserver](/ja/docs/reference/command-line-tools-reference/kube-apiserver/)および[kube-controller-manager](ja//docs/reference/command-line-tools-reference/kube-controller-manager/)、[kubelet](/ja/docs/reference/command-line-tools-reference/kubelet/)のコマンドラインで指定される`--cloud-provider`と`--cloud-config`の両方が必要です。
```
Comment (bells17) [inline content/ja/docs/concepts/cluster-administration/cloud-providers.md:45]:
```suggestion
典型的なインツリークラウドプロバイダーは、通常、[kube-apiserver](/ja/docs/reference/command-line-tools-reference/kube-apiserver/)および[kube-controller-manager](ja//docs/reference/command-line-tools-reference/kube-controller-manager/)、[kubelet](/ja/docs/reference/command-line-tools-reference/kubelet/)のコマンドラインで指定される`--cloud-provider`と`--cloud-config`の両方が必要です。
```

**Review by bells17 — COMMENTED** [review]
@nishipy レビューが遅れてしまい申し訳ありません。1点コメントしましたのでご確認をお願いします。

**Comment by nishipy** [comment]
ご指摘ありがとうございます

**Comment by bells17** [comment]
ご対応ありがとうございます🙏
/lgtm

**Pull request merged**